doc/reference/reference_sql/sql_user_guide.rst

```diff
@@ -808,10 +895,44 @@ value IS [NOT] NULL |br|
 CASE ... WHEN ... THEN ... ELSE ... END |br|
   ... for setting a series of conditions.
 
+.. _sql_map_expression:
+
+{ key : value } |br|
+... for MAP expressions. |br|
+Literal Examples: ``{'a':1}``, ``{ "column_1" : X'1234' }`` |br|
+Non-literal Examples: ``{"a":"a"}``, ``{UUID(), (SELECT 1) + 1}`` |br|
```

Comment:
I think here should be `{UUID(): (SELECT 1) + 1}`. Also, I think we should add some examples of MAP values where there is more than one key-value pairs, for example: `{1:'a123', 'two':uuid()}`.